packages/testlab/src/request.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {IncomingMessage} from 'http';
+import * as http from 'http';
+import * as https from 'https';
+import * as url from 'url';
+
+export function getAsync(urlString: string): Promise<IncomingMessage> {
```

Comment:
I think `httpGetAsync` may be a better name - more consistent with `httpsGetAsync` below. What do you think?